EIPS/eip-2228.md

```diff
@@ -0,0 +1,106 @@
+status: Draft
+type: Informational
+created: 2019-08-04
+---
+
+## Simple Summary
+
+The Ethereum network with chainid 1 is named Ethereum Mainnet.
+
+## Abstract
+
+The name for the Ethereum network with network ID 1 and chain ID 1 shall be Ethereum Mainnet or just Mainnet. This is a proper noun.
+
+This standard specifies the name for this network and provides reference examples in an effort to standardize the word choice and provide a common language for use to use to refer to this network.
+
+## Motivation
+
+The Ethereum network with chainid 1 is referenced from authoritative sources using several names that are conflicting:
+
+* Vitalik has called it "main net" at http://eips.ethereum.org/EIPS/eip-155 http://eips.ethereum.org/EIPS/eip-170
+* Vitalik has called it "mainnet" at http://eips.ethereum.org/EIPS/eip-86 
+* Vitalik has called it "Main net" at http://eips.ethereum.org/EIPS/eip-2* http://eips.ethereum.org/EIPS/eip-150
+   * \* In EIP-2, it is unclear if the first letter is capital because it is a proper noun or if it is the the first word in a text column presented in sentence case
+* Vitalik has called it "Mainnet" at http://eips.ethereum.org/EIPS/eip-7
+* Geth documentation calls its "mainnet" at https://geth.ethereum.org/support/FAQ
+* A final EIP here has referred to it as "Mainnet" at http://eips.ethereum.org/EIPS/eip-779
+
+## Specification
+
+The network name shall be Ethereum Mainnet, or just Mainnet if the context is known to be discussing Ethereum networks. This IS a proper noun. Several examples are given below which differentiate between usage of the name of the network versus a descriptive reference to the network.
+
+Any name or word styling (i.e. capitalization of the letters) of the network which is inconsistent with the test cases cited below shall NOT be used.
+
+## Rationale
+
+Choosing common word use promotes interoperability of implementations and increases customer awareness. Also, it adds a sense of professionalism when customers see the same word and word styling (i.e. capitalization of letters) across different implementations.
+
+Anybody that has travelled to certain countries and seen an "IPhone [sic]" repair store should immediately recognize that this is off-brand and unofficial. Likewise, the astute customer of Ethereum should recognize if they see the name of the network referred to using different names in different software, so let's avoid this.
+
+## Backwards Compatibility
+
+- MetaMask currently uses the wording "Main Ethereum Network" in the account network chooser. In this context, it is understood that "main" is used as an adjective, not a proper noun form. Such usage is acceptable per this EIP. Also, MetaMask is using "Main" and "Network" with a capitalized first letter for stylistic purposes, not to designate a proper noun.
+
+- Certain references to "Mainnet" and alternate spellings are published in EIPs made in the following published EIPs 2, 7, 150, 155, 170 and 779.
+
+## Test Cases
```

Comment:
I recommend renaming this to something more appropriate. "Test Cases" clearly refer to testing technical specifications, whereas this is a change in nomenclature. Something along the lines of "Usage" would be preferred.